feature-libs/checkout/components/components/shipping-address/shipping-address.component.html

```diff
@@ -34,6 +34,7 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
           >
             <cx-card
               [border]="true"
+              [index]="i"
```

Comment:
Should be transferred to both base and b2b

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0